src/api/youtubeApi.js

```diff
@@ -0,0 +1,210 @@
+              width: 480,
+              height: 360,
+            },
+          },
+          channelTitle: 'Funny Cats Life',
+          liveBroadcastContent: 'none',
+          publishTime: '2020-10-29T12:00:27Z',
+        },
+      },
+      {
+        kind: 'youtube#searchResult',
+        etag: 'XgUhdd2kF10KdWw828ZPV05lsuo',
+        id: {
+          kind: 'youtube#video',
+          videoId: 'vw0T0Ksx5ak',
+        },
+        snippet: {
+          publishedAt: '2020-11-27T17:58:21Z',
+          channelId: 'UCcChic4IF6tk-lmp9z-56sw',
+          title:
+            'Little Kitten &amp; Friends Adventure - Learn with a cute virtual cat Cutest Cat Best App for Kids',
+          description:
+            'Yay, Little Kitten is back and so excited about going to primary school! Our app „Little Kitten & Friends“ helps preparing children aged between 4 and 7 for their ...',
+          thumbnails: {
+            default: {
+              url: 'https://i.ytimg.com/vi/vw0T0Ksx5ak/default_live.jpg',
+              width: 120,
+              height: 90,
+            },
+            medium: {
+              url: 'https://i.ytimg.com/vi/vw0T0Ksx5ak/mqdefault_live.jpg',
+              width: 320,
+              height: 180,
+            },
+            high: {
+              url: 'https://i.ytimg.com/vi/vw0T0Ksx5ak/hqdefault_live.jpg',
+              width: 480,
+              height: 360,
+            },
+          },
+          channelTitle: 'iMilkin',
+          liveBroadcastContent: 'live',
+          publishTime: '2020-11-27T17:58:21Z',
+        },
+      },
+      {
+        kind: 'youtube#searchResult',
+        etag: 'iFP7AyXEON2V5vPYhfhzQjQmxLM',
+        id: {
+          kind: 'youtube#video',
+          videoId: 'DXUAyRRkI6k',
+        },
+        snippet: {
+          publishedAt: '2013-11-09T22:11:37Z',
+          channelId: 'UCVUdHi-tdW5AKdzMiTPG97Q',
+          title: 'Funny Cats and Kittens Meowing Compilation',
+          description:
+            'Here is a video of cats and kittens meowing to confuse your pets Puppies & Babies & Kitties OH MY! New videos all the time! Subscribe: ...',
+          thumbnails: {
+            default: {
+              url: 'https://i.ytimg.com/vi/DXUAyRRkI6k/default.jpg',
+              width: 120,
+              height: 90,
+            },
+            medium: {
+              url: 'https://i.ytimg.com/vi/DXUAyRRkI6k/mqdefault.jpg',
+              width: 320,
+              height: 180,
+            },
+            high: {
+              url: 'https://i.ytimg.com/vi/DXUAyRRkI6k/hqdefault.jpg',
+              width: 480,
+              height: 360,
+            },
+          },
+          channelTitle: 'funnyplox',
+          liveBroadcastContent: 'none',
+          publishTime: '2013-11-09T22:11:37Z',
+        },
+      },
+    ],
+  };
+}
+
+const maxResults = 25;
+
+export async function getVideos(searchItem) {
+  if (useMock) {
+    return mock().items;
+  }
+
```

Comment:
mocks should not be part of the prod code. Use them at tests only and it's better to keep them separated

Reply:
Mock was separated in other file and only is included for testing.